contentcuration/contentcuration/frontend/shared/views/channel/ChannelSharing.vue

```diff
@@ -83,6 +83,7 @@
         shareMode: SharingPermissions.EDIT,
         error: null,
         sharing: false,
+        reg: /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,24}))$/
```

Comment:
It is reasonably possible that this regex will prevent some valid email addresses being included.

I would suggest taking a more restrictive approach, in order to solve the example posted in the issue.

The principal issue was email addresses wrapped in `<a@a.com>` as `>` is not a valid character for the domain part of the email address, it is probably sufficient to add exclusion of that in the latter part of the email address, after `@`, to the regex.

Also, if you could add a test case for the `<a@a.com>` style email to the test suite: https://github.com/learningequality/studio/blob/hotfixes/contentcuration/contentcuration/frontend/shared/views/channel/__tests__/channelSharing.spec.js#L75 that would help to give some assurance that this fix resolves this issue.

Reply:
@rtibbles, I'm looking for one more decent approach to solve this issue, I'll update you once done if not I'll update you with the changes you suggested.
Thanks

Reply:
The simple approach is sufficient - the general consensus that I can see is that email validation is a hard problem, so we shouldn't try to find a one-size fix all solution, but rather prevent common user mistakes (which we will discover as people make them!)

Reply:
@rtibbles, You were right this approach is restricting some valid email addresses, I'll just add these `< >` and will update in the future.

Reply:
@rtibbles, Sorry to ping you again but I have a small doubt, I was playing around with the regex `else if (!/.+@.+\..+/.test(this.email))` so I removed `!` to check if it'll give an error but I noticed that it's not giving an error which means it was giving same results both the time(we remove or we don't `!`). Will you please confirm it?
Also, I usually get a bit confuse in the regex part will you please tell me where should I add `>` to `/.+@.+\..+/`
Thanks